.env.template

```diff
@@ -0,0 +1 @@
+POAP_API_KEY=___your_poap_api_key___
```

Comment:
Can we keep it without values, just as a template to fill? This way fails as required env var instead of failing for invalid value. Would be easier to debug.